src/sentry/consumers/__init__.py

```diff
@@ -396,12 +400,34 @@ def build_consumer_config(group_id: str):
             healthcheck_file_path, strategy_factory
         )
 
+    if dlq_topic is not None:
+        try:
+            cluster_setting = get_topic_definition(dlq_topic)
+        except ValueError:
+            click.BadParameter(
+                "Invalid DLQ topic: {dlq_topic}, all DLQ topics must have topic definition configured in settings"
+            )
+
+        producer_config = get_kafka_producer_cluster_options(cluster_setting)
+        producer_config.pop("compression.type", None)
+        producer_config.pop("message.max.bytes", None)
```

Comment:
Why are you removing these settings?

Reply:
I'm the other PRs I am referencing removes them
https://github.com/getsentry/sentry/pull/53019
https://github.com/getsentry/sentry/pull/49537
https://github.com/getsentry/sentry/pull/54647

Reply:
I think it's always better to produce with lz4 compression which I think you might be removing here

Reply:
Fixed, thanks